README.md

```diff
@@ -168,3 +190,72 @@ targets:
     dependencies:
       - "barback"
 ```
+
+#### Defining `Builder`s in your package (similar to transformers)
+
+If users of your package need to apply some code generation to their package,
+then you can define `Builder`s (from `package:build`) and have those be either
```

Comment:
maybe include a link here to the github repo for build? Or a link directly to build/README.md?